src/Roboteq_Protocol.h

```diff
@@ -0,0 +1,119 @@
+/**
```

Comment:
Can you write you're own class instead of copying stuff over? It can be heavily based off that driver, but stuff shouldn't just be copy pasted.